<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FullScreenConfigWidget</class>
 <widget class="QWidget" name="FullScreenConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>312</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Slideshow</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Interval:</string>
        </property>
        <property name="buddy">
         <cstring>mSlideShowIntervalSlider</cstring>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSlider" name="mSlideShowIntervalSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>60</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mSlideShowIntervalLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string notr="true">[N secs]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="mSlideShowLoopCheckBox">
        <property name="text">
         <string>Loop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mSlideShowRandomCheckBox">
        <property name="text">
         <string>Random</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Image Information</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="mConfigureDisplayedInformationButton">
        <property name="text">
         <string>Select Image Information to Display...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mThumbnailGroupBox">
     <property name="title">
      <string>Thumbnails</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="mShowThumbnailsCheckBox">
        <property name="text">
         <string>Show thumbnails</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Height:</string>
        </property>
        <property name="buddy">
         <cstring>mHeightSlider</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="mHeightSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>204</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>70</number>
        </property>
        <property name="maximum">
         <number>256</number>
        </property>
        <property name="value">
         <number>70</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mSlideShowIntervalSlider</tabstop>
  <tabstop>mSlideShowLoopCheckBox</tabstop>
  <tabstop>mSlideShowRandomCheckBox</tabstop>
  <tabstop>mConfigureDisplayedInformationButton</tabstop>
  <tabstop>mShowThumbnailsCheckBox</tabstop>
  <tabstop>mHeightSlider</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mShowThumbnailsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>mHeightSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mShowThumbnailsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
